lib/config.js

```diff
@@ -254,6 +255,7 @@ class Config {
 
           // HTTP conf
           secretToken: conf.secretToken,
+          apiKey: conf.apiKey,
           userAgent: userAgent,
```

Comment:
I see here we're setting `apiKey` in the configuration directly from the value set in the agent configuration.  This means we're expecting users to configure the base64 encoded value of the `id:key` pair (i.e. we don't base64 encode it for them) This is [the correct specified behavior](https://github.com/elastic/apm/issues/183).